database/schema.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import Column, Integer, String, ForeignKey
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+class StatsContainer(Base):
```

Comment:
i see no need for 3 tables here.   why have a "container id"?  what benefit is this?  when do we add new containers?

Reply:
what if I reduce this down to:

```
class StatsWorker(Base):
    __tablename__ = 'stats_worker'
    id = Column(Integer, primary_key=True, autoincrement=True)
    key = Column(String(255), unique=True)
    bad = Column(Integer)
    count = Column(Integer)
    
    


class StatsBin(Base):
    __tablename__ = 'stats_bin'

    id = Column(Integer, primary_key=True, autoincrement=True)
    stats_worker_id = Column(Integer, ForeignKey('stats_worker.id'))
    msize = Column(Integer)
    val = Column(Integer)
```

Reply:
there should only be one worker row per worker.   consider that we may need to use a key value store instead for performance

Reply:
Does the StatsWorker table not do that? As I see it each worker someone has would get a row there. So if a user has 10 workers all with the same msize theyd have 10 rows in that table all with the same msize and stats_container_id values, with differing id and val

Reply:
i see. the name of the table is very confusing.  the "container" table is one row per worker and the "worker" table is all the actual stats for each model size for the one worker.

workers:  key string,  count int, bad int
model_stats:  key string,  msize int, val double